internal/cephfs/volume.go

```diff
@@ -144,7 +152,17 @@ func createVolume(ctx context.Context, volOptions *volumeOptions, cr *util.Crede
 // subvolume. If the command is not available as a fallback it will use
 // CreateVolume to resize the subvolume.
 func resizeVolume(ctx context.Context, volOptions *volumeOptions, cr *util.Credentials, volID volumeID, bytesQuota int64) error {
-	if supported, ok := resizeSupportedList[volOptions.ClusterID]; supported || !ok {
+	// keyPresent checks whether corresponding clusterID key is present in clusterAdditionalInfo
+	var keyPresent bool
+	// verify if corresponding ClusterID key is present in the map,
+	// and if not, initialize with default values(false).
+	if _, keyPresent = clusterAdditionalInfo[volOptions.ClusterID]; !keyPresent {
+		clusterAdditionalInfo[volOptions.ClusterID] = &additionalClusterInfo{}
+	}
+
+	// resize subvolume when either it's supported, or when corresponding
+	// clusterID key is not present.
+	if supported := clusterAdditionalInfo[volOptions.ClusterID].resizeSupported; supported || !keyPresent {
```

Comment:
you can just use `clusterAdditionalInfo[volOptions.ClusterID].resizeSupported`, no need  to assigning it to `supported`